.github/ISSUE_TEMPLATE/DesignDoc.md

```diff
@@ -0,0 +1,28 @@
+---
+name: Design Doc tracking issue
+about: A standard template to follow when leading a project
+labels: Daily, NewFeature
+---
+
+## Proposal
+
+## Tasks
+
+- [ ] Post **Proposal (Problem/Solution)** in #expensify-open-source
+- [ ] Paste Proposal in the space above with a link to the Slack thread
+- [ ] Email `strategy@expensify.com` and paste in the Proposal
+- [ ] Fill out **High-level overview of the problem**, **Timeline**, and **Terminology** sections of the Design Doc
+- [ ] Email `strategy@expensify.com` (same email chain as before) with link to Design Doc
+- [ ] Host a _pre-design_ meeting in `#expensify-open-source` to discuss any necessary details in public before filling out the **High-level of proposed solution** section
```

Comment:
Should we link to an example here?

Reply:
Found a good one from Rory that I'm gonna add